tuna/miopen/driver/convolution.py

```diff
@@ -191,43 +206,43 @@ def set_defaults(self, defaults):
                          k, self.spatial_dim)
 
   @staticmethod
-  def get_params(tok1):
+  def get_params(tok1: str) -> Callable[[str, str], bool]:
     """Get full arg name"""
     return get_fd_name(tok1, TABLE_COLS_CONV_MAP)
 
   @staticmethod
-  def get_check_valid(tok1, tok2):
+  def get_check_valid(tok1: str, tok2: str) -> Callable[[str, str], bool]:
```

Comment:
This does not return a callable function. It returns the return of conv_arg_valid which is a boolean.